software-modules/internet/chromium/chromium.sh

```diff
@@ -20,12 +20,32 @@ TARGET_DIR="/run/initramfs/memory/system/huronOS/software/internet/"
 
 ## Install software
 apt update
-apt install --yes --no-install-recommends $NAME
+apt install --yes --no-install-recommends $NAME python-is-python3
 apt autoremove --yes
 
 ## Prepare final files
 cp ./$NAME.desktop /usr/share/applications/
 
+## Prepare bookmarks extension
+# The extension
+curl -o /usr/share/bookmark-bridge.crx -L https://github.com/huronOS/bookmark-bridge/releases/latest/download/bookmark-bridge.crx
+
+# The extension loader
+CHROMIUM_EXT_LOADER_PATH="/usr/share/chromium/extensions"
+mkdir -p $CHROMIUM_EXT_LOADER_PATH
+curl -o "$CHROMIUM_EXT_LOADER_PATH/cbfajpicdmpnnoijdmhdomaailmdaiim.json" https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/chrome-extension-loader.json
+
+# The extension's native host
+CHROMIUM_NATIVE_HOST="/etc/chromium/native-messaging-hosts"
+mkdir -p $CHROMIUM_NATIVE_HOST
+curl -o "$CHROMIUM_NATIVE_HOST/bookmarks_fetcher.json" https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/chrome-native-host.json
+
+# The middleman between huron and the browser
+curl -o /usr/bin/bookmarks-fetcher.py https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/app/bookmarks-fetcher.py
```

Comment:
Same, if we do depend on this files we should pack them on a ZIP release, oooor maybe we can create a directory in this repo `huronOS-build-tools/utils/` and create a github submodule pointing to the repo of bookmarks-bridge, this way we can copy directly from the submodule right from this repo.

Reply:
I like that idea, may be good to stick to that one, gotta learn how to use submodules, though hehe